### DISCUSSION TITLE
How to pass data between QuickAdd and Templater?

### DISCUSSION THREAD

**Discussion by dylane-tu** [chhoumann/quickadd#78] [Q&A]
I've got a Templater user script that retrieves a value (involving some user input and retrieval of notes from the vault -- i.e: Stuff that's not possible with just QuickAdd as far as I know) and inserts it into a template. I'd like to use that value in the title of a note I'm creating with QuickAdd that's run as part of a macro, after the Templater step.

TLDR: I've got a variable in Templater. Can I use it in a QuickAdd dynamic title?

**Comment by chhoumann** [comment]
Hey!

Unfortunately, QuickAdd runs Templater on a file independently of itself. Meaning, no sharing between them, etc.

However, if you write the variable to the Window object, fetch it with QuickAdd, and then write it to a variable, it's possible.

Depending on what you're doing, it's possible without any hacky solutions like that.
Have you seen the QuickAdd examples? User input and file manipulation is very possible with QuickAdd, but it just takes different steps than with Templater.
